modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -136,6 +136,13 @@ API assets, published by Exchange, are the components that comprise applications
 
 For details about sharing assets via a private Exchange or an Exchange public portal, see xref:exchange::about-sharing-assets.adoc[Share Assets].
 
+=== API Console
+
+When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and test your API specification. You can test RAML and OAS APIs when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.
```

Comment:
Same comment as above, maybe we could say _"expose and test"_.

Reply:
Done! Thanks!